pkg/translator/prometheusremotewrite/helper.go

```diff
@@ -455,10 +454,8 @@ func maxTimestamp(a, b pcommon.Timestamp) pcommon.Timestamp {
 
 // addSingleSummaryDataPoint converts pt to len(QuantileValues) + 2 samples.
 func addSingleSummaryDataPoint(pt pmetric.SummaryDataPoint, resource pcommon.Resource, metric pmetric.Metric, settings Settings,
-	tsMap map[string]*prompb.TimeSeries) {
+	tsMap map[string]*prompb.TimeSeries, baseName string) {
```

Comment:
As these are "helper" functions, I would suggest having `baseName` as an optional argument for these functions. 

If not specified by the caller, then it would be calculated as `prometheustranslator.BuildCompliantName(metric, settings.Namespace, settings.AddMetricSuffixes)`.

(this is just a suggestion, to keep the helper function as helpful as possible, and maintain backward compability)

Reply:
I'm inclined to think that making `baseName` optional falls into the YAGNI category, and just adding complexity with no practical gain, since these helper functions are only called from one site. For completeness, you'd have to e.g. write more test cases.

Reply:
I'm fine with it. Just to be sure: any chance these functions are being used outside of this project?

Reply:
> any chance these functions are being used outside of this project?

@bertysentry I would say no, considering they aren't exported from the package.